### PR TITLE
Sv feature/connecting

### PR DESCRIPTION
**Pull request by sebastianvera**
I added some improvements to the iot and command package, do not check the `main.go` since I'm just using it for testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ stop making usage of actionators as water pump, etc.
   },
   "meta": {
     "proccesses": [
-      { "name": "wisebot", "status": "running", "version": "1.0.1" },
-      { "name": "ble", "status": "updating", "version": "2.0.4" }
+      { "name": "wisebot", "status": "update:started", "version": "dfd09a7" },
```
Comment (cagodoy) [inline README.md:75]: wildcards 👀 👍 , are you sure that using the SHA as version is the best option?

File: README.md [hunk 2]
```diff
@@ -72,8 +72,8 @@ stop making usage of actionators as water pump, etc.
...
+      { "name": "ble", "status": "running:", "version": "5482f18" }
```
Comment (cagodoy) [inline README.md:76]: `running:started`?

File: command/command.go [hunk 3]
```diff
@@ -2,16 +2,65 @@ package command
...
 	"bufio"
+	"encoding/json"
 	"io"
 	"os"
 	"os/exec"
 )
 
+const (
+	statusIdle           = "idle"
+	statusRunning        = "running"
```
Comment (cagodoy) [inline command/command.go:13]: running `wisebot-device-manager` or `ble-service`?  how do you define what process or service you are referring with say `running`?

Comment (cagodoy) [reply command/command.go:13]: :in_monkey_spanish_mode:

ya entendí más o menos como estás resolviendo esta cosilla. así que omite el último comentario

**Review by cagodoy — APPROVED** [review]
Me gusta la onda, me gustaría probarlo con más detalle, pero paso a paso iremos descubriendo que es lo mejor. Asi que good job. 🚀

File: README.md (hunk 2 above)
Comment (sebastianvera) [reply README.md:76]: mmmm, I'm not sure, that's why we have an `error` and `done` and `stopped` ... neither `running:started` nor `running:stopped` makes sense to me :(

**Pull request merged**